src/binding.cpp

```diff
@@ -37,12 +37,12 @@ union Sass_Value* sass_custom_function(const union Sass_Value* s_args, Sass_Func
   }
 }
 
-void ExtractOptions(Local<Object> options, void* cptr, sass_context_wrapper* ctx_w, bool is_file, bool is_sync) {
-  NanScope();
+void ExtractOptions(v8::Local<v8::Object> options, void* cptr, sass_context_wrapper* ctx_w, bool is_file, bool is_sync) {
+  Nan::HandleScope scope;
 
   struct Sass_Context* ctx;
 
-  NanAssignPersistent(ctx_w->result, options->Get(NanNew("result"))->ToObject());
+  ctx_w->result.Reset(Nan::Get(options, Nan::New("result").ToLocalChecked()).ToLocalChecked()->ToObject());
```

Comment:
`ctx_w->result.Reset(Nan::To<v8::Object>(Nan::Get(options, Nan::New("result").ToLocalChecked()).ToLocalChecked()).ToLocalChecked());`